recipes/mesher/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set name = "mesher" %}
+{% set version = "2.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mesher-{{ version }}.tar.gz
+  sha256: a142b6c5773187ddd98bb9f422a9a8545f90f2482f4c9f5089b1f7f54ea30caf
+
+build:
+  entry_points:
+    - mesher = mesher:main
+    - mesh2shp = mesher.tools.mesh2shp:main
+    - mesh2vtu = mesher.tools.mesh2vtu:main
+    - mesh2merge = mesher.tools.meshmerge:main
+    - meshpermutation = mesher.tools.meshpermutation:main
+    - meshstats = mesher.tools.meshstats:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - make
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - scikit-build-core
+    - cmake
+    - pip
+    - cgal-cpp
+    - libboost-devel <3.8.5
+    - libgdal <3.8.5
+    - metis
+  run:
+    - python
+    - vtk
+    - numpy
+    - scipy
+    - matplotlib-base
+    - cloudpickle
+    - libgdal <3.8.5
+    - gdal <3.8.5
```

The rest should come in via `run_exports`
```suggestion
    - gdal
```